src/Microsoft.DotNet.Wpf/src/WpfGfx/codegen/mcg/generators/ProtocolFingerprint.cs

```diff
@@ -71,7 +71,7 @@ public override void Go()
             string generatedPath = 
                 Path.Combine(
                     _resourceModel.OutputDirectory,
-                    "src\\Graphics\\Include\\Generated"
+                    "src\\WpfGfx\\Include"
```

Comment:
Is it intentional ?

Reply:
Yes, it generates the files wgx_sdk_version.cs and wgx_sdk_version.h which are currently in `src\Microsoft.DotNet.Wpf\src\WpfGfx\include`. I included it in this PR because I wanted it to generate the files in the right path even if the content of the files needed to be fixed.

Reply:
The convention is that any generated file should be in a path with the parent directory as Generated, and as per that, the path mentioned here is correct. 

However, ProtocolFingerprint generator is not included in any .rsp file. So, actually it is not being used in any generation.
And as far as I can see, this file has not been generated since 2010.

I am trying to find if there was a modification in one of the .rsp files which removed this entry. ( These files have not been changed since 2010 as well )

Reply:
What your thought on this ? Shall we create an issue to track this ?

Reply:
Ok I just checked and the reason I changed this file is because originally, when I reversed engineered the .rsp files, I added ProtocolFingerprint.cs to the .rsp file but since I updated my PR with the original .rsp files that you sent it no longer hits this line.

For now, I think I'll just revert the removal of "\\Generated" since, as you said, every generated files should be in a "Generated" folder. I'll keep the changes of "Graphics" to "WpfGfx" since it's a change I did globally because "Graphics" was renamed to "WpfGfx" so it doesn't make sense to keep "Graphics" in the path, even if it's not used right now.

I'll also open an issue to track the current work and future work (Which will include checking if we want to keep ProtocolFingerprint.cs).